app/assets/stylesheets/sephora_style_guide/components/_tabs.scss

```diff
@@ -0,0 +1,135 @@
+select {
+  overflow: hidden;
+}
+
+.nav {
+  display: flex;
+  flex-wrap: wrap;
+  padding-left: 0;
+  margin-bottom: 0;
+  list-style: none;
+}
+
+.tab-content {
+  > .tab-pane {
+    display: none;
+  }
+
+  > .active {
+    display: block;
+  }
+}
+// Don't change the ones above ^^^^^^^^^^
+
+.nav-tabs {
+  display: block;
+  height: 45px;
+  white-space: nowrap;
+  flex-wrap: nowrap;
+  overflow: auto;
+  -webkit-overflow-scrolling: touch;
+  overflow-y: hidden;
+  
+  .nav-item {
+    display: inline-block;
+    white-space: normal;
+  }
+
+  .nav-link {
+    font-weight: 700;
+    letter-spacing: 0.5px;
+    padding: 0 15px;
+    display: block;
+    font-size: 14px;
+    line-height: 30px;
+    text-transform: uppercase;
+
+    &:hover {
+      color: $black;
+    }
+  }
+}
+
+.tab-content {
```

Comment:
As mentioned in chat, maybe move the styles to be under `.tab-content-default` so we can de-couple functionality from styles. Also consider merging some of the rules if appropriate.